flycheck.el

```diff
@@ -6872,6 +6873,22 @@ See URL `https://github.com/zaach/jsonlint'."
      (and (buffer-file-name)
           (string= (file-name-extension (buffer-file-name)) "json")))))
 
+(flycheck-define-checker json-pythonjson
+  "A JSON syntax checker using Python json.tool module.
+
+See URL `https://docs.python.org/3.5/library/json.html#command-line-interface'."
+  :command ("python" "-m" "json.tool" source-inplace null-device)
+  :error-patterns
+  ((error line-start
+          (message) ": line " line " column " column (one-or-more not-newline)
```

Comment:
it matchs the char of the error, e.g: ` (char 57)`. I was not sure if included or not, but given that all the `error-patterns` include the `line-end` regex, I preferred to add it